src/features/vote/helpers/formatVoteDataToCommit.ts

```diff
@@ -88,5 +113,7 @@ export async function formatVoteDataToCommit(
     })
   );
 
+  localStorage.setItem("backupCommits", JSON.stringify(newCommits));
```

Comment:
again not sure if its possible, but to return `newCommits` and do this in the caller

Reply:
it could be done that way, although I'm not sure it would simplify it.

Maybe it would be better to just have a parallel function that is run and is atomic.